shell/config/settings.ts

```diff
@@ -258,8 +259,19 @@ export const DEFAULT_PERF_SETTING: PerfSettings = {
             generic: true,
           }
         }
+      },
+      management: {
+        resources: {
+          enableAll:  false,
+          enableSome: {
+            enabled: [
+              { resource: CAPI.RANCHER_CLUSTER, context: ['home', 'side-bar'] },
+              { resource: MANAGEMENT.CLUSTER, context: ['side-bar'] },
+            ],
+            generic: false,
+          }
+        }
```

Comment:
Are we splitting SSP up into different user settings?

To me it seems like we should really just make SSP an all or nothing setting.

Reply:
This was important for the gradual conversion to SSP, possibly still so given enabling in a global way might start breaking things (including e2e).

It's all store basedall, some (specific resources, generic